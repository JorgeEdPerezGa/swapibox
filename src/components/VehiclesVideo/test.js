import React from 'react';
import VehicleVideo from './index';
import setupTest from '../../setupTests';
import { shallow } from 'enzyme';

describe('<VehicleVideo/>', () => {
  const wrapper = <VehicleVideo />;

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
