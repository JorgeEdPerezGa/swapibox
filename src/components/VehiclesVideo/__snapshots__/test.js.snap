// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VehicleVideo/> should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <VehiclesVideo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <video
        autoPlay={true}
        className="background-video"
        loop={true}
        muted={true}
        playsInline={true}
        poster="vehicles.mp4"
>
        <source
                src="vehicles.mp4"
                type="video/mp4"
        />
</video>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "autoPlay": true,
        "children": <source
          src="vehicles.mp4"
          type="video/mp4"
/>,
        "className": "background-video",
        "loop": true,
        "muted": true,
        "playsInline": true,
        "poster": "vehicles.mp4",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "src": "vehicles.mp4",
          "type": "video/mp4",
        },
        "ref": null,
        "rendered": null,
        "type": "source",
      },
      "type": "video",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <video
          autoPlay={true}
          className="background-video"
          loop={true}
          muted={true}
          playsInline={true}
          poster="vehicles.mp4"
>
          <source
                    src="vehicles.mp4"
                    type="video/mp4"
          />
</video>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "autoPlay": true,
          "children": <source
            src="vehicles.mp4"
            type="video/mp4"
/>,
          "className": "background-video",
          "loop": true,
          "muted": true,
          "playsInline": true,
          "poster": "vehicles.mp4",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "src": "vehicles.mp4",
            "type": "video/mp4",
          },
          "ref": null,
          "rendered": null,
          "type": "source",
        },
        "type": "video",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
